File: src/main/java/edu/kit/iti/formal/stvs/StvsVersion.java
package edu.kit.iti.formal.stvs;

/**
 * @author Alexander Weigl
 * @version 1 (15.04.17)
 */
public class StvsVersion {
    public static String getVersion() {
        return StvsVersion.class.getPackage().getImplementationVersion();
    }

    public static String getBuildId() {
        return StvsVersion.class.getPackage().getSpecificationVersion();
    }

    public static String getName() {
        return StvsVersion.class.getPackage().getName();
    }

}

File: stvs/src/main/kotlin/edu/kit/iti/formal/stvs/StvsVersion.kt
package edu.kit.iti.formal.stvs

/**
 * @author Alexander Weigl
 * @version 1 (15.04.17)
 */
object StvsVersion {
    @JvmStatic
    val version: String
        get() = StvsVersion::class.java.getPackage().implementationVersion

    @JvmStatic
    val buildId: String
        get() = StvsVersion::class.java.getPackage().specificationVersion
    @JvmStatic
    val name: String
        get() = StvsVersion::class.java.getPackage().name

    @JvmStatic
    val windowTitle: String
        get() = "Structured Text Verification Studio - STVS"
}
